ouroboros-consensus/test/consensus-test/Test/Consensus/MiniProtocol/LocalStateQuery/Server.hs

```diff
@@ -78,25 +78,29 @@ tests = testGroup "LocalStateQueryServer"
 --   chain, a state and send the 'QueryLedgerTip'. Collect these results.
 -- * Check that when acquiring failed, it rightfully failed. Otherwise, check
 --   whether the returned tip matches the block.
-prop_localStateQueryServer
-  :: SecurityParam
+prop_localStateQueryServer ::
+     SecurityParam
   -> BlockTree
   -> Permutation
-  -> Positive (Small Int)
   -> Property
-prop_localStateQueryServer k bt p (Positive (Small n)) = checkOutcome k chain actualOutcome
+prop_localStateQueryServer k bt p = checkOutcome k chain actualOutcome
   where
     chain :: Chain TestBlock
     chain = treePreferredChain bt
 
-    points :: [Target (Point TestBlock)]
-    points = permute p $
-         replicate n VolatileTip
-      ++ (SpecificPoint . blockPoint <$> treeToBlocks bt)
+    -- A random sequence of targets: one for each block in the tree and also a
+    -- random number of immtip/voltip queries
```

Comment:
> A random sequence of targets: one for each block in the tree and also **a random number of immtip/voltip queries**

Isn't the change that is explained in the PR description to exactly *remove* the fact that there is a random number of voltip queries, instead just using one?